crates/swc_ecma_loader/src/resolvers/exports.rs

```diff
@@ -90,20 +96,22 @@ fn candidate_matches<'a>(candidate: &str, rel_path: &'a str) -> Option<Match<'a>
 
 #[derive(Debug, PartialEq, Eq)]
 enum Subpath {
-    Target(String),
+    Targets(Vec<String>),
     Conditions(BTreeMap<String, Subpath>),
     Exclude,
 }
 
 enum SubpathMatch<'a> {
-    Target(&'a str),
+    Targets(Vec<&'a str>),
```

Comment:
Could we make this `Targets(&'a [&'a str])` or is there something preventing that?

Reply:
I couldnt find a way of doing that, since we have a `Vec<String>` that we are converting from in `Subpath`, we need to map over the items and collect then to convert it to `Vec<&str>`.

We would have to make Subpath::Targets a `Vec<&str>` or something, but then we need to introduce lifetimes on the exports struct, and packagejson, not sure if its worth going down that route, if you dont have any other ideas? :thinking:

Reply:
updated to slice